src/medCore/process/medAbstractProcess.h

```diff
@@ -32,6 +32,8 @@ class MEDCORE_EXPORT medAbstractProcess : public dtkAbstractProcess
     medAbstractProcess( medAbstractProcess * parent = NULL );
     virtual ~medAbstractProcess();
 
+    using dtkAbstractProcess::setInput;
```

Comment:
I see what you did here, but i'm not even sure that it will fix your probleme (you have no guarantee that your process use a setInput methode taking a type deriving from dtkAbstractData*) so maybe you will be to call the methode but it will do nothing.

Your probleme should be fix when the process layer will be design, you wil bel then able to use something like:

```cpp
medAbstractWhateverKindOfProcess *p = medProcess::whateverKindOfProcess::pluginFactory().create("whateverKindOfProcess_id");
```
where `medAbstractWhateverKindOfProcess` will expose the setter / accessor that you need to set the inputs or get the outputs for this specific kind of process.